projects/packages/my-jetpack/_inc/components/product-interstitial/jetpack-ai/product-page.jsx

```diff
@@ -64,10 +64,11 @@ export default function () {
 		nextTier,
 		usagePeriod: usage,
 		isOverLimit,
+		tierPlansEnabled,
 	} = aiAssistantFeature || {};
 
-	const hasUnlimited = currentTier?.value === 1;
 	const isFree = currentTier?.value === 0;
+	const hasUnlimited = currentTier?.value === 1 || ( ! tierPlansEnabled && ! isFree );
```

Comment:
Is this to consider tiered plans as unlimited?

Reply:
Yes.

I think it might be debatable, but that's what I was talking about today: do not enforce tier plan limits once tier plans are disabled, just consider them "unlimited". In this context (My Jetpack) that's all we need to know (ensure proper card and product page).

Mind noting this PR doesn't address some cases just yet, such as being over the fair usage limit (both on card and product page).